src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+/**
+ * A customer is a person who visits a shop for buying goods.
+ *
+ * The customer can have a loyaltyCard that has a specific number.
+ */
+public final class Customer {
+
+    private final LoyaltyCard loyaltyCard;
+
+    /**
+     * Constructor for determination the customer's loyalty card number.
+     *
+     * @param loyaltyCard
+     */
+    public Customer(LoyaltyCard loyaltyCard) {
```

Comment:
There's no need to repeat a type name in a parameter name. Just `card` would be fine.